src/scripts/demo.mjs

```diff
@@ -142,6 +194,7 @@ const MOCK_COMFY = (function () {
 		Init(__username, __auth, channelNames) {
 			if (channelNames.length) {
 				broadcaster.user = channelNames[0];
+				BROADCASTER_NAME = broadcaster.user;
```

Comment:
same question.
Can the already existing `broadcaster` be reused instead of this extra variable?
By lifting that variable creation outside of the `COMFY_MOCK` and reusing it inside.